src/lib/api/api.go

```diff
@@ -523,19 +523,34 @@ func (a *API) DeleteConfigRoutes(w http.ResponseWriter, r *http.Request) {
 	if err := decoder.Decode(&req); err != nil {
 		panic(err)
 	}
+	var updated bool
+	updated = false
```

Comment:
`found := false`

Reply:
This is a Backend correction. In case of BenchCLI it will be very helpful.
As in case user is using CLI he will pass the Route or RouteID to delete the route from Config file and incase if it isn't present in server(spelling mistake while making request to server) will return the error message that the route you entered is not present and user will identify his mistake and again make a proper delete request the server. If in case this error is not handled user will get message that route has been deleted even though the route wasn't deleted.
In case of UI there is no issue.